lms/envs/production.py

```diff
@@ -1138,10 +1138,16 @@
 )
 
 # Custom urls in navbar
+COLARAZ_SUB_DOMAIN = ENV_TOKENS.get(
+    'COLARAZ_SUB_DOMAIN',
+    COLARAZ_SUB_DOMAIN
+)
 COLARAZ_NAVIGATION_URLS = ENV_TOKENS.get(
     'COLARAZ_NAVIGATION_URLS'
 )
-
+COLARAZ_APP_LINKS_API = ENV_TOKENS.get(
```

Comment:
```suggestion
COLARAZ_APP_API_LINKS = ENV_TOKENS.get(
```

Reply:
@imhassantariq basically it's an API endpoint which returns LINKS, so kindly suggest accordingly :).

Reply:
`COLARAZ_APP_LINKS_API_ENDPOINT` or `COLARAZ_APP_LINKS_API_URL`

Reply:
Updated